packages/components/src/Banner/Banner.tsx

```diff
@@ -74,6 +74,22 @@ Banner.Message = function BannerMessage(props: { children?: React.ReactNode }) {
 
 /**
  * A banner used to provide and highlight information to a user or ask for a decision or action.
+ *
+ * ```tsx
+ * <Banner
+ *   onDismiss={handleDismiss}
+ *   textContent={
+ *     <>
+ *       <Banner.Title>{bannerTitle}</Banner.Title>
+ *       <Banner.Message>{bannerMessage}</Banner.Message>
+ *     </>
+ *   }
+ * />
+ * ```
+ *
+ * Please note the code snippets under each Banner story on storybook are misleading
+ * because they use <BannerTitle> and <BannerMessage> when they are actually called
+ * <Banner.Title> and <Banner.Message>.
```

Comment:
could you try
```suggestion
 * because they use `<BannerTitle>` and `<BannerMessage>` when they are actually called
 * `<Banner.Title>` and `<Banner.Message>`.
```
? rendering weird in the screenshot

Reply:
Yes, thanks!